plugins/woocommerce/changelog/fix-remote-inbox-total-payments-value-doc

```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: tweak
+Comment: Remote Inbox Notifications documentation is primarily used internally, since the notifications come from WooCommerce.com
```

Comment:
This changelog `Comment: ` style threw me for a second, but I see other changelog entries like that as well, so I guess both styles are acceptable.

Reply:
@joshuatf This changelog `Comment` is the comment that gets logged when you run the changelogger tool and note that you want to skip a changelog entry. So, it's not really the changelog entry, but rather a comment explaining why you are skipping the changelog entry. Confusing, certainly.